Parkinsons-Mouse-tutorial.ipynb

```diff
@@ -0,0 +1,3062 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parkinson’s Mouse Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This guide assumes you have QIIME 2 installed (e.g. using this procedure). To execute the script properly, open this notebook in a Jupyter Notebook from within a conda QIIME 2 environment.\n",
+    "\n",
+    "This tutorial is an adaptation of the same tutorial that may be found on the official QIIME 2 docs website. The original tutorial uses the QIIME 2 CLI interface.\n",
```

Comment:
Sugestia:  W pierwszych 4 akapitach, pojawiają się stwierdzenia w stylu 
"QIIME 2 docs website", "conda QIIME2 interface", "QIIME 2 CLI interface", "Artifact API", 
"moving pictures tutorial" - brak tam łączy do referencji. Oczywiście można zostawić 
użytkownikowi zadanie żeby sam się dowiedział, gdzie i co to jest, ale lepiej nie utrudniać
pracy żółtodziobom.